conkit/core/contactmap.py

```diff
@@ -394,6 +394,29 @@ def singletons(self):
                 singletons.remove(contact.id)
         return singletons
 
+    @property
+    def highest_contact(self):
+        """The :obj:`~conkit.core.contact.Contact` with the highest residue sequence number in the :obj:`~conkit.core.contactmap.ContactMap`
+
+        Returns
+        -------
+        :obj:`~conkit.core.contact.Contact`
+           A :obj:`~conkit.core.contact.Contact` object
+
+        """
+        if len(self) == 0:
+            return None
+
+        highest_index = 0
+        highest_id = None
+
+        for contactid in self.as_list():
+            if contactid[0] > highest_index or contactid[1] > highest_index:
```

Comment:
`contactid[1]` is strictly equal to or greater than `contactid[0]` since res1 <= res2